examples/server/server.cpp

```diff
@@ -118,6 +96,7 @@ struct slot_params {
 
     std::vector<std::string> antiprompt;
     bool timings_per_token = false;
+    bool ignore_eos = false;
```

Comment:
With new models the `ignore_eos` functionality is losing relevance. There are now many different "end-of-generation" tokens and it's not just a single EOS token anymore. We should remove this logic and only support logit biases, which is more general. Just a note, no need to do it in this PR.